### PR TITLE
Right panel data

### PR DESCRIPTION
**Pull request by jaredhirsch**
ugh, after getting sucked into frustration-brain trying to get `jQuery.trigger` to not swallow my custom event data, fell back to DOM APIs and got it (barely) working:

I've got to bail very shortly, but will do what I can to clean this mess up before I head out ^_^

### CODE REVIEW

**Comment by jaredhirsch** [comment]
well, ran out of time. idk why embedly fails to return basic stuff like the favicon in some cases, but I think that's the cause of the busted img tags. I didn't get a chance to set the template bits to only render if, say, result.favicon is truthy.

File: scripts/main.js [hunk 1]
```diff
@@ -162,10 +167,95 @@ var AutocompleteSearchResultsView = Backbone.View.extend({
...
+
+  initialize: function() {
+    window.addEventListener('selectionChanged', this.onSelectionChanged.bind(this));
+
+    // XXX this might be a mistake, do we even know if the page continues to
+    //     run after the popup is dismissed?
+    $(document).on('visibilitychange', this.onVisibilityChanged.bind(this));
+  },
+
+  onSelectionChanged: function($item) {
```
Comment (jaredhirsch) [inline scripts/main.js:235]: Derp. An event is passed in here, so it'll never be falsy.

**Comment by jaredhirsch** [comment]
Hmm, probably should also hide the old item as soon as the selection changes. Currently the old content just sits till newer content arrives.

**Comment by nchapman** [comment]
> Hmm, probably should also hide the old item as soon as the selection changes. Currently the old content just sits till newer content arrives.

I did this, and it still isn't quite right. We should probably make it so that the pane goes blank (or has a spinner) instead of hiding when we expect data to be there. Right now it hides every time but feels weird when it takes a while for the content to come in.

**Comment by nchapman** [comment]
I think this is good enough to merge for now. :+1:

**Pull request merged**